requirements.in

```diff
@@ -1,42 +1,42 @@
 #-----------------------------------
 #   FSD Utils
 #-----------------------------------
-funding-service-design-utils>=1.0.22,<1.1.0
+funding-service-design-utils
 requests
 
 #-----------------------------------
 #   Flask Version
 #-----------------------------------
-Flask==2.1.1
+Flask
 
 #-----------------------------------
 #   Security
 #-----------------------------------
-flask-talisman==0.8.1
+flask-talisman
 
 #-----------------------------------
 #   Static Assets
 #-----------------------------------
-Flask-Assets==2.0
-Flask-Compress==1.10.1
-cssmin==0.2.0
-pyScss==1.4.0
-jsmin==3.0.1
-govuk-frontend-jinja==2.0.0
+Flask-Assets
+Flask-Compress
+cssmin
+pyScss
+jsmin
+govuk-frontend-jinja
```

Comment:
Might be desirable to restrict these, at least by major version, else we risk arbitrarily breaking APIs.

Reply:
Unless I'm misunderstanding I think pip-compile prevents constant changes to our otherwise locked reqs.txt (whenever they do change it will be obvious in the PR) and our tests will catch any breaking changes when changed.

Reply:
Yes, it does but I guess that assumes our tests will catch all issues!

I suppose we can try this approach and see if we get a lot of regressions due to major changes in dependencies.

Reply:
True, it seems one of those trade-offs. I suppose tests should be catching those things, so at least it would highlight a gap in our testing, but it's not an ideal way to do it by any stretch! 

Shall we try this approach, monitor and make the change to specify major versions should we see the regressions?